PR #21 Reformatted (#128)
mongodb/docs-meta: source/style-guide/markup/format/roles.txt

=====
Roles
=====

> Roles are a type of directive that are used inline and can be used to create

Reviewer: ```suggestion
Roles are a type of directive that you can use inline to create
```

> Roles are a type of directive that are used inline and can be used to create

Reviewer 2: [suggestion for minimalism]

Could we say:

"Roles are inline directives used to create cross-references or apply formatting."

Author: I'm fine with the change, but i wonder whether "inline directive" might cause ambiguity as to whether it's a term or just a directive that has inline placement.

Reviewer 2: That's a fair point. "inline directives" might be a bit too convoluted / jargony. I'm happy to leave this as-is as even though it is a bit wordier, it may be easier to understand than my suggestion.

cross-references or apply formatting. Use them consistently to ensure
uniform formatting.

> Learn more about roles from the
> `Sphinx documentation <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`__.

Reviewer: ```suggestion
To learn more about roles, see the
`Sphinx documentation <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`__.
```
